src/app/components/ContactUs.tsx

```diff
@@ -42,8 +43,28 @@ export default function ContactUs() {
         body: JSON.stringify(data),
       });
       reset();
+
+      toast.success("Your message has been sent successfully!", {
+        position: "top-center",
+        autoClose: 3000,
+        hideProgressBar: false,
+        closeOnClick: true,
+        pauseOnHover: true,
+        draggable: true,
+        progress: undefined,
+      });
```

Comment:
Since the ToastContainer configuration (position, autoClose, etc.) is already defined in layout file, you can just call toast.success("Message") – it will automatically use those settings. No need to repeat the same rules every time. 😊
```suggestion
      toast.success("Your message has been sent successfully!");
```